documentation/design-docs/dotnet-tools.md

```diff
@@ -396,6 +397,63 @@ CONVERT
       Writing:       ./trace.speedscope.json
       Conversion complete
 
+
+CONVERT
+
+    dotnet-trace stack -p|--process-id <pid>
+                       [-h|--help]
+
+    Prints the stack for every thread in the target process
+
+    -h, --help
+        Show command line help
+
+    Examples:
+      > dotnet-trace stack -p 1234
+      Stack for Thread (2207382):
+          UNMANAGED_CODE_TIME
```

Comment:
This looks weird in the output. Should we replace this with [Native Frames]? Or somethings alike?

Reply:
This is output from TraceEvent's symbolicating logic.  It should be fairly simple to just check for this text and replace it before printing.